websites/A/AniMixPlay/presence.ts

```diff
@@ -0,0 +1,106 @@
+const presence = new Presence({
+    clientId: "787739407720513596"
+}),
+
+strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused",
+    browsing: "presence.activity.browsing"
+});
+
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  const startTime = Date.now(),
+    endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
+const browsingTimer = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const presenceData: presenceData = {
+        largeImageKey: "logo-v2"
+    };
+
+    if (document.location.hostname == "animixplay.to") {
+      if (document.location.pathname == "/") {
+
+        urlParams = new URLSearchParams(window.location.search);
+
+        if (urlParams.get("q")) {
+          presenceData.startTimestamp = browsingTimer;
+          presenceData.details = "Currently searching for...";
+          presenceData.state = '"' + urlParams.get("q") + '"';
+          presenceData.smallImageKey = "browsing-v1";
+          presenceData.smallImageText = (await strings).browsing;
+        } else if (urlParams.get("genre")) {
+            presenceData.startTimestamp = browsingTimer;
+            presenceData.details = "Currently exploring...";
+            presenceData.state = "" + urlParams.get("genre").toLowerCase().replaceAll(",", " + ") + " related anime";
+            presenceData.smallImageKey = "browsing-v1";
+            presenceData.smallImageText = "Exploring...";
+        } else if (urlParams.get("season") && urlParams.get("year")) {
+            presenceData.startTimestamp = browsingTimer;
+            presenceData.details = "Currently exploring...";
+            presenceData.state = "" + urlParams.get("season").toLowerCase() + " " + urlParams.get("year").toLowerCase() + " anime";
+            presenceData.smallImageKey = "browsing-v1";
+            presenceData.smallImageText = "Exploring...";
+        } else {
+            presenceData.startTimestamp = browsingTimer;
+            presenceData.details = "Currently browsing...";
+            presenceData.smallImageKey = "browsing-v1";
+            presenceData.smallImageText = (await strings).browsing;
+        }
+      } else if (document.location.pathname.includes("/v4/") || document.location.pathname.includes("/v1/") || document.location.pathname.includes("/v7/")) {
```

Comment:
better use regex to match all stream pages url ( /v1/ , /v2/ , /v3/ , ... , /v9/ )
`/^\/v.\//.test(document.location.pathname)`
or just add match for url with other number.

Reply:
Added